library/LuaApi.cpp

```diff
@@ -158,6 +158,50 @@ void Lua::GetVector(lua_State *state, std::vector<std::string> &pvec)
     }
 }
 
+void Lua::PushInterfaceKeys(lua_State *L,
```

Comment:
I think this should go in `LuaTools.cpp`. Not really sure why `PushPosXY[Z]` wound up in this file, but most other `Push*` functions are in `LuaTools.cpp` or inline in `LuaTools.h`. I think `LuaApi.cpp` is more for Lua-facing functions.

Reply:
done. I moved the other Lua:: methods as well.